src/service/http-service.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+    baseURL: 'https://scrabble-workout.firebaseio.com',
```

Comment:
yes, but I mentioned elsewhere (messenger) that I cannot upload it on the server untill I don't merge with the master branch. so - if you don't mind - I'll push it directly to the master, hm? (otherwise I don't know if it even works)

Reply:
I think you should be able to make it work without pushing words json file to master before the current PR; in other words: async fetching of words and words.json in public folder should be introduced together in the same PR

please try using **relative path** instead of an absolute one

and be sure you rebase onto the most recent master